Neos.Flow/Classes/Mvc/DispatchComponent.php

```diff
@@ -98,12 +98,11 @@ protected function prepareActionRequest(ComponentContext $componentContext)
         $arguments = $httpRequest->getArguments();
 
         $parsedBody = $this->parseRequestBody($httpRequest);
-        if ($parsedBody !== []) {
+        $httpRequest = $httpRequest->withParsedBody($parsedBody);
```

Comment:
```suggestion
        if ($parsedBody !== null) {
            $httpRequest = $httpRequest->withParsedBody($parsedBody);
        }
```
Or maybe not if it is `null`